src/mtool/api/test/Mtool_Env-Loki-server.postman_environment.json

```diff
@@ -12,6 +12,16 @@
 			"value": "5000",
 			"enabled": true
 		},
+		{
+			"key": "telemetryIP",
+			"value": "107.108.83.97",
```

Comment:
Please change the IP to generic. Maybe you can use 127.0.0.1

Reply:
File name indicates loki server. should I also change host to generic one.

Reply:
Either you can change the file name. Or you can change the value to the hostname loki,